wiki/Disambiguation/EZ/en.md

```diff
@@ -0,0 +1,9 @@
+# EZ
+
+**EZ** may also refer to:
+
+-   The [Easy game modifier](/wiki/Game_Modifiers/#easy).
+-   The version name of a beatmap.
+-   The difficulty of a beatmap:
+    -   ![](/wiki/shared/diff/easy-m.png) EZ, for osu!mania.
```

Comment:
add the rest of the game modes too

Reply:
I don't think when someone writes EZ, they think taiko or catch, but alright.